Core/FileLoader.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  FileLoader.swift
+//  DuckDuckGo
+//
+//  Created by Mia Alexiou on 17/05/2017.
+//  Copyright © 2017 DuckDuckGo. All rights reserved.
+//
+
+import Foundation
+
+enum FileError: Error {
+    case unknownFile
+    case invalidFileContents
+}
+
+class FileLoader {
+
+    func load(name: String, ext: String) throws -> Data {
```

Comment:
I don't think you need to change this, but if you want to parse the extension off, and just pass a file name like:

```swift
let contents = FileLoader().load('happygolucky.json');
```